source_docs/installation/installation.md

```diff
@@ -4,13 +4,13 @@ Installing Bolt
 This page explains the various ways of installing Bolt. You can either use the
 command-line or your FTP-client to install it.
 
-There are several methods to install Bolt, but we recommend the "easiest way"
+There are several methods to install Bolt, but we recommend **the "easiest way"**
 [using the server command-line](install-command-line).
```

Comment:
None of the three points below is called "easiest"… maybe we should either:
  * change "easiest" to "traditional"
  * change "traditional" to "easiest"